### PR TITLE
Fe 3 add common pages

### PR DESCRIPTION
**Pull request by AntoninaKobzar**
pages added

### CODE REVIEW

**Comment by AntoninaKobzar** [comment]
pages folder`s been removed

File: app/company_profile/page.tsx
Comment (alexskursky) [inline app/company_profile/page.tsx:1]: This page should have dynamic path

File: app/user_profile/page.tsx
Comment (alexskursky) [inline app/user_profile/page.tsx:1]: This page should have dynamic path

**Review by alexskursky — CHANGES_REQUESTED** [review]
1. Group your routes with `Route Groups`. Create groups for `auth` and `main app` (private and public routes).
https://nextjs.org/docs/app/building-your-application/routing/route-groups
2. Name your components files with a capital letter.
3. Group your routes like this - `users/[userId]`, `companies/[companyId]` etc.

**Comment by AntoninaKobzar** [comment]
routes with Route Groups created for auth and main app (private and public routes).
components files named with a capital letter.
routes grouped :users/[userId], companies/[companyId] etc.

**Review by alexskursky — APPROVED** [review]

**Pull request merged**